frontend/src/components/question-creation/question-elements/modals/multiple-choice/EditMultipleChoiceModal.tsx

```diff
@@ -38,6 +40,7 @@ const EditMultipleChoiceModal = ({
   return (
     <MultipleChoiceModal
       data={data}
+      isMultiSelect={isMultiSelect}
       isOpen={isOpen}
       onClose={closeModal}
       onConfirm={updateMultipleChoiceElement}
```

Comment:
same as above! if we could rename this modal to `EditMultiDataModal.tsx` or something, that would be awesome! and the folder should probably be renamed to `multi-option` as well!